src/pages/dashboard/Dashboard.tsx

```diff
@@ -27,18 +28,31 @@ const StyledLinks = styled.div`
 
 const StyledSearchBarContainer = styled.div`
   grid-area: search-bar;
-  width: 35rem;
+  width: 80%;
   @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
     width: 15rem;
   }
 `;
 
 const Dashboard: FC = () => {
   const { t } = useTranslation();
+  const history = useHistory();
+  const searchTerm = new URLSearchParams(history.location.search).get('query');
```

Comment:
denne trengs vel ikke? hvis vi har `query` skal det vel uansett være til `/search` og ikke relevant for `Dashboard`(?)

Reply:
sant!